src/handbook/peopleops/compensation.md

```diff
@@ -73,4 +71,7 @@ awarded to be expensed for a dinner out. The bonus is awarded per release.
 
 ### Health Care
 
-FlowForge offers a health care package for all employees in the United States, Canada, and India.
+FlowForge offers a health care package for all employees in the United States, Canada, and India. 
+
+For employees located in the United States, Deel offers assistance with basic health insurance coverage which includes dental and vision benefits. However, employees who choose not to participate in the health insurance provided by FlowForge have the option to opt out. This can be done by registering with Zenefits and declining coverage during the enrollment process on the platform. It is important to note that if an employee decides to opt out, they must have alternative qualified coverage to ensure compliance with the ACA (Affordable Care Act).
```

Comment:
```suggestion
For employees located in the United States, FlowForge offer basic health insurance coverage which includes dental and vision benefits through Deel.
However, employees who choose not to participate in the health insurance provided by FlowForge have the option to opt out. This can be done by registering with Zenefits and declining coverage during the enrollment process on the platform. It is important to note that if an employee decides to opt out, they must have alternative qualified coverage to ensure compliance with the ACA (Affordable Care Act).
```